docs/source/includes/_translations.md

```diff
@@ -0,0 +1,178 @@
+            "href": "/field_guides/{translations.field_guide}",
+            "type": "field_guides"
+        },
+        "translations.tutorial": {
+            "href": "/tutorials/{translations.tutorial}",
+            "type": "tutorials"
+        },
+        "translations.workflow": {
+            "href": "/workflows/{translations.workflow}",
+            "type": "workflows"
+        }
+    }
+}
+
+```
+
+foo
+## `strings` and `string_versions`
+
+Whenever a collaborator changes translatable resources, Panoptes will copy over
+those strings into the "primary" translation resource. This is the translation
+resource that has its language set to the language of the translatable resource
+(which it will typically inherit from the Project it belongs to, where
+applicable). This primary translation resource is not editable via the API.
+
+Upon copying data into the primary translation resource, Panoptes calculates a
+version number for the translated resource. This version number will be
+applicable to the strings that were changed. For instance, if a project
+translation is initially this:
+
+<div class="center-column"></div>
+```json
+{
+  "strings": {"title": "Hello", "description": "This needs to be written"},
+  "string_versions": {"title": 1, "description": 1}
+}
+```
+
+And the project owner updates the description (via the [Projects](#projects)
+API) but not the title, once Panoptes finishes updating the translation (which
+happens asynchronously), the translation resource will look like this:
+
+<div class="center-column"></div>
+```json
+{
+  "strings": {"title": "Hello", "description": "This is now a real description"},
+  "string_versions": {"title": 1, "description": 2}
+}
+```
+
+Any translation editing UI should use this to provide the following features:
+
+1. If for instance the string version of `description` on a secondary
+   translation is set to `1`, while the primary translation has a version number
+   of `2` or higher for this field, in the UI this field should be marked as
+   outdated to let translators know that the project owner has updated the
+   description and they need to recheck and update the translation for this
+   field.
+0. When a translator updates a field, the UI should send along the number for
+   that field's string version of the primary translation, as shown to the
+   translator. That is to say, if a translator then updates the `description`
+   field of the secondary translation, when the editing UI calls the Panoptes
+   API to save the updated translation, it should send along the value `2` for
+   the `string_versions["description"]`. It should only update the values for
+   strings that were actually touched, so as no to reset the outdatedness
+   information for fields that weren't updated yet.
+
+
+## List translations
+
+## Get a single translation
+
+## Create a translation
+
+## Update a translation
+
+## Publish the latest translation
+
+```http
+POST /api/translations/123/publish HTTP/1.1
```

Comment:
Noting here that https://github.com/zooniverse/json-api-client doesn't support POST to URLs like this, which is a pain.

Reply:
What would be your preferable alternative?

Reply:
I think we're going to have to update the client. My reading of the client code is you can't POST, only PUT, to a resource that's specified by ID, but I could be looking at the wrong part of the client code.

I'll have a look at how the client handles links. From memory, they use URLs like this, I think. I might need help from someone who's more familiar with the details of the API than I am.